src/Form/BecomeAdherentType.php

```diff
@@ -19,12 +20,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('conditions', CheckboxType::class, [
                 'required' => false,
             ])
+            ->add('address', AddressType::class, ['label' => false])
```

Comment:
Unless we have very good reason, we should not need `label` option at this point as we have the possibility to not render the label in twig.

Reply:
If I don t do that , form_end will render the label

Reply:
Use `{{ form_end(form, {render_rest: false}) }}` in twig then.

Reply:
but we remove all hidden fields not rendered :s like csrf token